plugins/localfile.js

```diff
@@ -8,7 +8,8 @@ var fs = require('fs');
 var mime = require('mime')
 
 var isFile = function(item) {
-  return fs.existsSync(item.path) && fs.statSync(item.path).isFile();
+  const stat = fs.statSync(item.path);
+  return fs.existsSync(item.path) && fsStat.isFile() && fsStat.size > 0;
```

Comment:
why call `.statSync` if `.existsSync` will fail? instead of writing one big oneliner, use `return` statement to return early.

also, as the rest of the code uses `var`, use that instead of `url` to keep existing language level compatibility.